### ISSUE TITLE
Add view permissions to the access scheme

### ISSUE THREAD

**Issue by parijke** [palantirnet/workbench_access#128]
Forgive me if someone asked before, I could not find it in the issue list.

Beside node create/edit restrictions, is it possible to restrict vieweing access to the nodes assigned to the sections? Like it was with Organic Groups, you could type a piece of content as private (viewable for section members) or public (viewable for everybody).

If not, do you hevae a suggestion that adds to Workbench Access as it is a very nice way to group content.

**Comment by agentrickard** [comment]
I did a standalone module for this in D7, which would be the proper way here.

https://www.drupal.org/sandbox/agentrickard/1878184

**Comment by ghost** [comment]
Hello 👋 I am working on a project in D8 and was wondering how I might limit a user's Create, View, Edit, and Delete to their respective section in the hierarchy. Thank you.

**Comment by agentrickard** [comment]
Create, Edit, and Delete are handled by the main Workbench Access module.

Workbench Access deliberately does not handle the View case.

**Comment by ghost** [comment]
Yes! Thank you. I made incorrect assumptions regarding Workbench Access independence from Workbench. As soon as I required/enabled what I was looking for, worked. 😅 Thank you!
